README.rst

```diff
@@ -125,17 +125,29 @@ An example module.config.js file looks like the following.  You can copy this in
      frontend-platform, for instance, you need to run `npm run build` in the repo before module.config.js will work.
      */
      localModules: [
-       { moduleName: '@edx/brand', dir: '../brand-openedx' }, // replace with your brand checkout
-       { moduleName: '@edx/paragon/scss/core', dir: '../paragon', dist: 'scss/core' },
-       { moduleName: '@edx/paragon/icons', dir: '../paragon', dist: 'icons' },
-       { moduleName: '@edx/paragon', dir: '../paragon', dist: 'dist' },
-       { moduleName: '@edx/frontend-platform', dir: '../frontend-platform', dist: 'dist' },
+       { moduleName: '@edx/brand', dir: '../src/brand-openedx' }, // replace with your brand checkout
+       { moduleName: '@edx/paragon/scss/core', dir: '../src/paragon', dist: 'scss/core' },
+       { moduleName: '@edx/paragon/icons', dir: '../src/paragon', dist: 'icons' },
+       { moduleName: '@edx/paragon', dir: '../src/paragon', dist: 'dist' },
+       { moduleName: '@edx/frontend-platform', dir: '../src/frontend-platform', dist: 'dist' },
      ],
    };
 
-Note that the "dir" and "dist" keys give you granular control over the shape of your repository's distribution.  Paragon, for instance, needs two separate entries to pick up both JS and SCSS imports.
+Steps
+~~~~~
 
-This mechanism uses Webpack resolve aliases, as documented here: https://webpack.js.org/configuration/resolve/#resolvealias
+#. Run ``npm install && npm run build`` within any shared NPM package you want to use locally.
```

Comment:
This note is sort of duplicative with the "IMPORTANT NOTE" above... I think with the addition of this Steps section, we can move the note down here and remove the three lines on 123-125.

Reply:
Agreed, and done.